src/BuildingRegistry.Projections.Wfs/Migrations/20230825093350_AddSpatialIndexesV2.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace BuildingRegistry.Projections.Wfs.Migrations
+{
+    public partial class AddSpatialIndexesV2 : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql(@"
+	            ALTER TABLE [wfs].[buildingsV2]
+		            ADD CalculatedGeometry AS (geometry::STGeomFromWKB([Geometry], 31370)) PERSISTED
+	            GO");
+
+            migrationBuilder.Sql(@"
+	            CREATE SPATIAL INDEX [SPATIAL_GebouwV2_Geometrie] ON [wfs].[buildingsV2] ([CalculatedGeometry])
```

Comment:
`[Geometry]`

Reply:
copy pasta from wms. did not notice it was already Geometry